src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,51 @@
+.nav {
+  height: 60px;
+
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    height: 60px;
+    line-height: 60px;
+    list-style: none;
+  }
+
+  &__item {
+    margin-left: 20px;
+    &:first-child {
+      margin: 0;
+    }
+  }
+
+  &__link {
+    display: block;
+    height: 60px;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $main-text-color;
+    font-size: 12px;
+    transition: all $hover-duration;
+    font-weight: 500;
+    position: relative;
+
+    &:hover {
+      color: $accent-color;
+    }
+
+    &.is-active {
+      color: $accent-color;
```

Comment:
The active link color uses `$accent-color`. Ensure that this variable is defined and correctly imported in your SCSS setup.